uint32 command
uint32 mClientId
uint32 UID
uint32 mnId
float64[3] mWorldPos

uint32 nNextId
int32 mnFirstKFid
int32 mnFirstFrame
int32 nObs

Observation[] mObservations
uint8[32] mDescriptor